WordPress/WordPressUITests/Tests/EditorGutenbergTests.swift

```diff
@@ -25,8 +27,8 @@ class EditorGutenbergTests: XCTestCase {
 
     func testTextPostPublish() throws {
 
-        let title = getRandomPhrase()
-        let content = getRandomContent()
+        let title = "Text post title"
+        let content = "Text post content"
```

Comment:
Random values in tests are great to catch weird behavior and input edge cases that the test author might not have considered at the time of writing.

But, one needs a way to repeat the test with the same input in order to reproduce the issue. Otherwise, we risk getting flaky tests with no way of knowing why.

Without a configurable seed from which to generate random values, random input can be dangerous for the consistency of the test suite.

Another reason I'm making this change is because the Simulator takes a long time to type such long inputs one character at a time. _That's the reason why I ended up looking at this code in the first place._

Reply:
> Without a configurable seed from which to generate random values, random input can be dangerous for the consistency of the test suite.

This is a good point, thanks for the extra context!

> because the Simulator takes a long time to type such long inputs one character at a time

So far I'm using iPhone 12 with iOS 14.5 as a workaround, but actually typing is the only thing it does faster. From what I saw recently, the execution of tests that do not involve typing is noticeably faster on iPhone 13.